### PR TITLE
To npz

### PR DESCRIPTION
**Pull request by kayibal**
Just a distributed version of to_npz.

### CODE REVIEW

File: sparsity/dask/io.py [hunk 1]
```diff
@@ -132,4 +146,36 @@ def _npz_read_divisions(paths, level=None):
                 file=paths[i], div1=divisions[i], div2=divisions[i+1]
             ))
 
-    return divisions
\ No newline at end of file
+    return divisions
+
+
+def write_npz_metadata(writes, divisions, paths, fn,
+                       block_size, storage_options):
```
Comment (michcio1234) [inline sparsity/dask/io.py]: Looks like `writes` is unused.

File: sparsity/dask/io.py [hunk 2]
```diff
@@ -132,4 +146,36 @@ def _npz_read_divisions(paths, level=None):
...
+    data = {}
+    data['divisions'] = np.asarray(divisions)
+    data['partitions'] = np.asarray(paths)
+
+    _write_dict_npz(data, fn, block_size, storage_options)
+
+
+def to_npz(sf: SparseFrame, path: str, block_size=None,
+           storage_options=None, compute=True):
+    if not '*' in path:
```
Comment (michcio1234) [inline sparsity/dask/io.py]: Should be `if '*' not in path`.

File: sparsity/test/test_dask_sparse_frame.py [hunk 3]
```diff
@@ -205,11 +205,23 @@ def test_read_npz():
         sf.iloc[50:75].to_npz(os.path.join(tmp, '3'))
         sf.iloc[75:].to_npz(os.path.join(tmp, '4'))
 
-        dsf = dsp.read_npz(os.path.join(tmp, '*.npz'))
+        dsf = dsp.read_npz(os.path.join(tmp, '*.npz'), sorted=True)
```
Comment (michcio1234) [inline sparsity/test/test_dask_sparse_frame.py]: Does this test pass? Didn't you remove `sorted` kwarg?

File: sparsity/dask/io.py (hunk 1 above)
Comment (kayibal) [reply sparsity/dask/io.py]: It is unused but necessary for how I use this function with dask. We might pass metadata of writing individual files via this parameter later...

**Pull request merged**